ruby/ext/google/protobuf_c/glue.c

```diff
@@ -19,3 +19,35 @@ google_protobuf_FileDescriptorProto* FileDescriptorProto_parse(
   return google_protobuf_FileDescriptorProto_parse(serialized_file_proto,
                                                    length, arena);
 }
+
+char* EnumDescriptor_serialized_options(const upb_EnumDef* enumdef, size_t *size) {
+  const google_protobuf_EnumOptions* opts = upb_EnumDef_Options(enumdef);
```

Comment:
@haberman - is this pattern safe? Is it OK to return from this method the return value from `google_protobuf_EnumOptions_serialize()`? Is it a pointer into memory that was potentially just freed by the call to `upb_Arena_Free()`?

Reply:
It is unsafe as written. You'd want to pass the arena in as a parameter